package-lock.json

```diff
@@ -0,0 +1,7881 @@
+{
```

Comment:
i'm using yarn for this package - we dont want to include a yarn.lock in addition to a package-lock

https://classic.yarnpkg.com/lang/en/

Reply:
that's cool. I'll change it. But why do people use yarn? I never understood how that is better than npm. Node ships out of the box with npm and it has always worked for me. I'll make the amendment tho

Reply:
good question - within the last year or so npm has caught up to a lot of features that yarn was providing that npm wasn't, such as it was generally faster and provided some API improvements. ive got used to using it for that reason and it feels a bit nicer to use, i dont have strong opinions on not changing it but dont think it should be in scope here and not worth changing unless there's a good reason to